zb-xiaomi.js

```diff
@@ -188,7 +189,7 @@ const MODEL_IDS = {
         clusterId: CLUSTER_ID.ILLUMINANCE_MEASUREMENT,
         attr: 'measuredValue',
         value: 0,
-        parseValueFromAttr: 'parseIlluminanceMeasurementAttr',
+        parseValueFromAttr: 'parseNumericAttr',
       },
```

Comment:
It isn't clear why you changed this. The Spec says that the number presented in measureValue isn't lux but rather a forumla which parseIllumincanceMeasurementAttr does.

Reply:
if you use parseIlluminanceMeasurementAttr you get an strange number 1.xxxx something certainly not lux :-) Now you get a number which is 0 on dark and gets higher the more light you shine on it. Not sure if this is lux.

Reply:
zigbee: zb-00158d000302cf27-motion property: illuminance profileId: 0104 endpoint: 1 clusterId: 0400 report value: 126

In the logging you can see that last value in this case 126 varies with light intensity.

Reply:
OK - I'll leave it like it is for now. I ordered an RTCGQ11LM and when it arrives I'll compare it other devices and see it the numbers make any sense.

Reply:
Flashlight:
zigbee: zb-00158d000302cf27-motion property: illuminance profileId: 0104 endpoint: 1 clusterId: 0400 report value: 713